components/eamxx/tests/coupled/physics_only/shoc_p3_nudging/input_nudging.yaml

```diff
@@ -11,6 +11,8 @@ time_stepping:
 atmosphere_processes:
   schedule_type: Sequential
   atm_procs_list: [shoc,p3,nudging]
+  p3:
+    max_total_ni: 740.0e3
```

Comment:
Any reason not to try another value in all the unit tests? It's quite unlikely that we'd trigger a bug, but we may as well. Something that may trigger an edge case may be interesting to test though... Like what happens if we use 1e0 or something small? Will certain special path of p3 get triggered more/less?

Reply:
@crterai or @hassanbeydoun , any thoughts on a good value to use here other than 740?  Could we try something super small like 1.0 as Luca suggests?